modules/datastore/modules/datastore_mysql_import/src/Service/MysqlImport.php

```diff
@@ -299,19 +311,21 @@ protected function truncateHeader(string $column): string {
    *   List of CSV headers.
    * @param string $eol
    *   End Of Line character for file importation.
+   * @param int $header_line_count
+   *   Number of lines occupied by the csv header row.
    *
    * @return string
    *   Generated SQL file import statement.
    */
-  private function getSqlStatement(string $file_path, string $tablename, array $headers, string $eol): string {
+  protected function getSqlStatement(string $file_path, string $tablename, array $headers, string $eol, int $header_line_count): string {
     return implode(' ', [
```

Comment:
```suggestion
  protected function getSqlStatement(string $file_path, array $headers, string $eol, int $header_line_count): string {
    $tablename = $this->dataStorage->getTableName();
    return implode(' ', [
```
CodeClimate is complaining about number of args, and there's no reason we can't do this here.

Reply:
I would prefer not to do this, since it would make the `getSqlStatement()` method no longer be deterministic (pure/static), and would further complicate the already convoluted MySQL service mock in `MysqlImportTest::getMysqlImporter()`.

I understand the idea behind CodeClimate's 4 parameter limit, but I think string builder functions (like this one) and constructor methods are really an exception.

If this is really bothering you, I'd be open to possibly using a parameter object to clean up the method signature a bit.